From d44682457e7bf24ece67d79dfb5f1a11dd66ab92 Mon Sep 17 00:00:00 2001
From: GitHub Action <action@github.com>
Date: Wed, 7 Jan 2026 05:48:56 +0000
Subject: [PATCH] feat: Squash PR #16619 changes

---
 docker/rocm.Dockerfile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docker/rocm.Dockerfile b/docker/rocm.Dockerfile
index 4c3269fa7..c4e90d258 100644
--- a/docker/rocm.Dockerfile
+++ b/docker/rocm.Dockerfile
@@ -31,7 +31,7 @@ ENV BUILD_TRITON="0"
 ENV BUILD_LLVM="0"
 ENV BUILD_AITER_ALL="1"
 ENV BUILD_MOONCAKE="1"
-ENV AITER_COMMIT="v0.1.7.post5"
+ENV AITER_COMMIT="3b0ef17043742d47d4a5d35cef8e205152edf1fc"
 ENV NO_DEPS_FLAG=""
 
 # ===============================
@@ -42,7 +42,7 @@ ENV BUILD_TRITON="0"
 ENV BUILD_LLVM="0"
 ENV BUILD_AITER_ALL="0"
 ENV BUILD_MOONCAKE="1"
-ENV AITER_COMMIT="v0.1.7.post5"
+ENV AITER_COMMIT="3b0ef17043742d47d4a5d35cef8e205152edf1fc"
 ENV NO_DEPS_FLAG=""
 # ===============================
 # Chosen arch and args
-- 
2.52.0

